internal/runtime/init/init.go

```diff
@@ -109,6 +109,22 @@ type Descriptor struct {
 	Path string
 }
 
+// removeDuplicatePaths removes duplicate file paths in order
+func removeDuplicatePaths(paths []string) []string {
+	pathMap := make(map[string]bool)
```

Comment:
nit: this can simply be `map[string]struct{}` -- the boolean is unnecessary. Below you'd say

```
pathMap[path] = struct{}{}
```

Reply:
Good idea, I changed it